website/cue/reference/components/sources/base/fluent.cue

```diff
@@ -22,10 +22,7 @@ base: components: sources: fluent: configuration: {
 	address: {
 		description: "The address to listen for connections on."
 		required:    true
-		type: {
-			number: {}
-			string: {}
-		}
+		type: string: examples: ["0.0.0.0:9000", "systemd", "systemd#3"]
```

Comment:
🤔 does `fluent` support the systemd driven address?

Reply:
Short answer- yes.

It's a TcpSource and `try_bind_tcp_listener` accepts both enum variants. The docs for `fluent` don't correctly outline this. But, to avoid having to make this PR for four total sources 😭 , I elected not to make each of the sources it already is touching, pristine.

Reply:
😮 ok TIL!

Reply:
Duuude. I was way off here.

The code compiles, like, the explanation I gave above is not wrong in that sense.

But I think behaviorally, it doesn't make sense at all that `fluent` or `logstash` would use a file descriptor 😭 
IDK what I was thinking.

I think the `SocketListenAddr` was wrongly used. Unless I'm missing something, that's a bug. I can address here.

Reply:
I think I interpreted the question about the vector fluent source, not about the fluent integration 🤦

Reply:
Now you have me confused 😆

Reply:
I inquired about this offline and, apparently it's valid to use socket activation for both fluent and logstash.

So, I'll revert my recent changes and add a comment.